packaging/torcharrow/meta.yaml

```diff
@@ -12,7 +12,7 @@ requirements:
   host:
     - python
     - setuptools
-    {{ environ.get('CONDA_PYTORCH_BUILD_CONSTRAINT') }}
+    - pytorch
```

Comment:
If not, why not just remove `pytorch` from the dependency?

Reply:
yeah good point, I copied it from the other recipes, but now that I think of it, building probably doesn't need pytorch. Let me test it.